scripts/gitlab/publish-snap.sh

```diff
@@ -3,13 +3,13 @@
 set -e # fail on any error
 set -u # treat unset variables as error
 
-TRACK=`awk -F '=' '/^track/ {print $2}' ./util/version/Cargo.toml`
+TRACK=`awk -F '=' '/^track/ {print $2}' ../../util/version/Cargo.toml`
```

Comment:
- fixed the choice of the track
- the path should start from the pkg source since `.gitlab-ci.yml` launches this file there.